docusaurus/docs/React/utility-components/channel-preview-ui.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: channel_preview_ui
+sidebar_position: 4
+title: ChannelPreview UI
+---
+
+The ChannelPreview UI component receives props via the [`ChannelPreview`](./channel-preview.mdx) wrapper, and
+these props are a combination of the `ChannelPreview` props with additional UI specific items. The wrapper also handles the required logic
+while the UI component is responsible for the UI display and the click functionality for channel selection. 
+
+This UI component is set via the `Preview` prop on `ChannelList`, and if nothing is provided then the default is used, [`ChannelPreviewMessenger`](https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelPreview/ChannelPreviewMessenger.tsx).
+
+## Basic Usage
+
+This UI component is very customizable; below is an example of how to use the `Preview` prop, and for a more involved step-by-step guide, using the many provided props,
+see [ChannelPreview Code Example](../custom-code-examples/channel_list_preview#the-preview-prop-component).
+
+```tsx
+const YourCustomChannelPreview = (props) => {
+  // render custom preview info here
+};
+
+<ChannelList
+  Preview={(previewProps) => <YourCustomChannelPreview {...previewProps} />}
+/>;
+```
+
+## Props
+
+### channel
+
+This required prop is the `channel` to be previewed; comes from either the `channelRenderFilterFn` or `usePaginatedChannels` call from
+[ChannelList](https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelList/ChannelList.tsx) and does not need to be set manually.
+
+| Type    |
+| ------- |
+| Channel |
+
+### active
+
+If the component's channel is the active (selected) Channel.
+
+| Type    |
+| ------- |
+| boolean |
+
+### activeChannel
+
+The currently selected channel object.
+
+| Type    |
+| ------- |
+| Channel |
+
+### Avatar
+
+The custom UI component to display the an avatar for the `channel`.
+
+| Type      | Default                                                                                               |
+| --------- | ----------------------------------------------------------------------------------------------------- |
+| component | [Avatar](https://github.com/GetStream/stream-chat-react/blob/master/src/components/Avatar/Avatar.tsx) |
+
+### channelUpdateCount
+
+A number that forces the update of the preview component on channel update.
+
+| Type   |
+| ------ |
+| number |
+
+
+### displayImage
+
+Image of Channel to display.
```

Comment:
let's lower case all the Channel instances to channel in the props section since they're not talking about the component, but the object